pipelines/resources/deploy-ztp-edgeclusters-minimal.yaml

```diff
@@ -0,0 +1,291 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: deploy-ztp-edgeclusters-minimal
```

Comment:
Should we call this `deploy-ztp-edgecluster-no-mirror` ?

Reply:
updated to deploy-ztp-edgecluster-no-mirror